src/background/screenshot.ts

```diff
@@ -22,7 +22,7 @@ function dataURItoBlob(dataURI: string): Blob {
   return new Blob([ab], { type: mimeString })
 }
 
-export async function takeScreenshot(tab: browser.tabs.Tab, tabs: typeof browser.tabs, dispatchBackgroundActions: DispatchBackgroundActions): Promise<void> {
+export async function takeScreenshot(tab: TabInfo, tabs: typeof browser.tabs, dispatchBackgroundActions: DispatchBackgroundActions): Promise<void> {
```

Comment:
`tab.url` can be undefined here. We need to validate it.

Reply:
Since we're using the `tabs` permission in our manifest.json, we [should have the URL](https://developer.chrome.com/extensions/tabs)